Packs/Vectra_AI/Integrations/VectraDetect/VectraDetect.py

```diff
@@ -737,9 +737,17 @@ def extract_detection_data(detection: Dict[str, Any]) -> Dict[str, Any]:
     source_host = detection.get('src_host')
     source_host_id = source_host.get('id') if source_host else None
 
+    summary = detection.get('summary')
+    if summary:
+        description = summary.get('description')
+        dst_ips = summary.get('dst_ips')
+        dst_ports = summary.get('dst_ports')
```

Comment:
```suggestion
        description = summary.get('description', "")
        dst_ips = summary.get('dst_ips', "")
        dst_ports = summary.get('dst_ports', "")
```

Reply:
Hi @YuvHayun, so you prefer to have an empty string than a None value. I'll change that

Reply:
Done